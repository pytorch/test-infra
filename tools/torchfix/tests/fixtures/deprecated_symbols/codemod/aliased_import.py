import torch
import torch as foo
import torch as bar
import torch as baz

A = torch.arange(9.0).reshape(3, 3)
A3 = foo.chain_matmul(A, A, A)
rc1 = bar.cholesky(torch.mm(A3.t(), A3))
rc2 = baz.qr(torch.mm(A.t(), A))

