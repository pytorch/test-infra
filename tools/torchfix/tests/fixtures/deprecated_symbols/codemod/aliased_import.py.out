import torch
import torch as foo
import torch as bar
import torch as baz

A = torch.arange(9.0).reshape(3, 3)
A3 = foo.linalg.multi_dot([A, A, A])
rc1 = bar.linalg.cholesky(torch.mm(A3.t(), A3))
rc2 = baz.linalg.qr(torch.mm(A.t(), A))

